### PR TITLE
add Telegraf 1.13.2 release notes

### PR DESCRIPTION
**Pull request by kelseiv**

### CODE REVIEW

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ menu:
     parent: About the project
 ---
 
+## v1.13.2 [2020-01-21]
+
+### Bug fixes
+
+- Warn without error when processes input is started on Windows.
```
Comment (kelseiv) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: `processes` input plugin?

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ menu:
...
+- Only parse certificate blocks in `x509_cert` input plugin.
+- Add custom attributes for all resource types in `vsphere` input plugin.
+- Support URL agent address form with udp in `snmp` input plugin.
+- Record device fields when attributes is `false`.
```
Comment (kelseiv) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: Device fields in any plugin?

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 3]
```diff
@@ -8,6 +8,18 @@ menu:
...
+- Remove invalid timestamps from Kafka messages.
+- Fix `json_strict` option and set `default` to `true`.
```
Comment (kelseiv) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: Does this apply to any configuration?

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md (hunk 1 above)
Comment (sanderson) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]:
~~~
- Warn without error when Processes (`processes`) input plugin is started on Windows.
~~~

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 4]
```diff
@@ -8,6 +8,18 @@ menu:
     parent: About the project
 ---
 
+## v1.13.2 [2020-01-21]
+
+### Bug fixes
+
+- Warn without error when processes input is started on Windows.
+- Only parse certificate blocks in `x509_cert` input plugin.
```
Comment (sanderson) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]:
~~~
- Only parse certificate blocks in X.509 Certificate (`x509_cert`) input plugin.

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 5]
```diff
@@ -8,6 +8,18 @@ menu:
...
+- Add custom attributes for all resource types in `vsphere` input plugin.
```
Comment (sanderson) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]:
~~~
- Add custom attributes for all resource types in VMware vSphere (`vsphere`) input plugin.
~~~

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md [hunk 6]
```diff
@@ -8,6 +8,18 @@ menu:
...
+- Support URL agent address form with udp in `snmp` input plugin.
```
Comment (sanderson) [inline content/telegraf/v1.13/about_the_project/release-notes-changelog.md]:
~~~
- Support URL agent address form with UPD in SNMP `snmp` input plugin.
~~~

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md (hunk 2 above)
Comment (danielnelson) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: this is the `smart` input plugin

File: content/telegraf/v1.13/about_the_project/release-notes-changelog.md (hunk 3 above)
Comment (danielnelson) [reply content/telegraf/v1.13/about_the_project/release-notes-changelog.md]: `json` parser

**Pull request merged**